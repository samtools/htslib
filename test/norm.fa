>20 20:1339000-1339300
AGGATGGGGCTCATTAATAGAGCTCCACTTGTCTCCAGAATCACTGGTGAGGAAGGGGAG
TGTTGCCCCCACATTCGTGCACAGCAGGGATGGTTCACCGAACTCCACACCAGTCTCTGC
AGAGCCTGTTGGGGAGAGGAGGGCTGTGGTTTCTTTGATGGTGTTCACCTGGAGTAGAGC
AAGTATTGTCAAAAGGGTCATCCTCGGAGGTTGCAGTGAGCCGAGATCGCACCATTGCAC
TGCAGCCTGGGAGACAGAGCAAGACTCCATCTCAAAAAAAAAAAAAAAAAAAAAGGCCAT
C
>1 1:10143-10443
CTAACCCCTAACCCTAACCCTAACCCTAACCCTAACCTAACCCTAACCCTAACCCTAACC
CTAACCCTAACCCTAACCCTAACCCTAACCCCTAACCCTAACCCTAAACCCTAAACCCTA
ACCCTAACCCTAACCCTAACCCTAACCCCAACCCCAACCCCAACCCCAACCCCAACCCCA
ACCCTAACCCCTAACCCTAACCCTAACCCTACCCTAACCCTAACCCTAACCCTAACCCTA
ACCCTAACCCCTAACCCCTAACCCTAACCCTAACCCTAACCCTAACCCTAACCCTAACCC
>2 1:1382388-1382602
GGGCGTCTCATAGCTGGAGCAATGGCGAGCGCCTGGACAAGGGAGGGGAAGGGGTTCTTA
TTACTGACGCGGGTAGCCCCTACTGCTGTGTGGTTCCCCTATTTTTTTTTTTTTCTTTTT
GAGACGGAGTCTCGCTCTGTCACCCAGGCTGGAGTGCAGTGGCACAATCTCGGCTCACTG
CAAGCTCCACCTCCTGGGTTCACGCCATTCTCCTG
