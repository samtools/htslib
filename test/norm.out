##FILTER=<ID=PASS,Description="All filters passed">
##fileformat=VCFv4.1
##INFO=<ID=INDEL,Number=0,Type=Flag,Description="Indicates that the variant is an INDEL.">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##FORMAT=<ID=PL,Number=G,Type=Integer,Description="Phred-scaled likelihood">
##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Depth">
##contig=<ID=1,length=2147483647>
##contig=<ID=2,length=2147483647>
##contig=<ID=20,length=2147483647>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	XY00001
1	105	.	TAAACCCTA	T,TAACCCTA	999	PASS	INDEL	GT:PL:DP	1/2:0,3,5,3,5,5,3,5,3,5:1
2	101	.	A	ATT	999	PASS	INDEL	GT:DP	1/2:1
2	101	.	ATTTTTTTTTTTTT	ATTTTTTTTTTTTTTC,ATTTTTTTTTTTTTT	999	PASS	INDEL	GT:DP	1/2:1
2	115	.	C	T	999	PASS	INDEL	GT:DP	1/2:1
20	5	.	TGGG	TAC,TG,TGGGG,AC	999	PASS	INDEL	GT:PL:DP	1/2:0,3,5,3,5,5,3,5,3,5:1
20	273	.	C	CAA,CAAA	999	PASS	INDEL	GT:PL:DP	1/2:0,3,5,3,5,5:1
20	273	.	C	CAAAAAAAAAA	999	PASS	INDEL	GT:PL:DP	./.:0,0,0:0
20	273	.	C	CAA	999	PASS	INDEL	GT:PL:DP	./.:0,0,0:0
20	275	.	A	C	999	PASS	INDEL	GT:PL:DP	0/1:0,0,0:0
